guides/micronaut-security-jwt/groovy/src/test/groovy/example/micronaut/AppClient.groovy

```diff
@@ -10,13 +10,16 @@ import io.micronaut.http.client.annotation.Client
 import io.micronaut.security.authentication.UsernamePasswordCredentials
 import io.micronaut.security.token.jwt.render.BearerAccessRefreshToken
 
+import static io.micronaut.http.MediaType.TEXT_PLAIN
+
+@CompileStatic
 @Client("/")
 interface AppClient {
 
     @Post("/login")
     BearerAccessRefreshToken login(@Body UsernamePasswordCredentials credentials)
 
-    @Consumes(MediaType.TEXT_PLAIN)
+    @Consumes(TEXT_PLAIN)
     @Get
-    String home(@Header String authorization)
+    String home(@Header('authorization') String authorization)
```

Comment:
Is this necessary ? If you want to be clear about the header name which will be used, we should use the constant in `HttpHeaders`.

Reply:
this was to be consistent with the Java and Kotlin code

Reply:
Could we use 

`@Header String authorization`

or `@Header(HttpHeaders.AUTHORIZATION) String authorization` ?

I prefer the latter.